src/main/java/frc/robot/motorcontrol/HallEffectMagnet.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.motorcontrol;
+
+import edu.wpi.first.math.util.Units;
+import frc.robot.subsystems.TiltedElevatorSubsystem.ElevatorState;
+
+public class HallEffectMagnet {
+
+    private double extendDistanceMeters;
+    private ElevatorState elevatorState;
+
+    public HallEffectMagnet(double extendDistanceMeters) {
+        this(extendDistanceMeters, null);
+    }
+
+    public HallEffectMagnet(double extendDistanceMeters, ElevatorState elevatorState) {
+        this.extendDistanceMeters = extendDistanceMeters;
+        this.elevatorState = elevatorState;
+    }
+    
+    public double getExtendDistanceMeters() { return extendDistanceMeters; }
+    
+    public double getExtendDistanceInches() { return Units.metersToInches(extendDistanceMeters); }
+
+    public boolean hasElevatorState() { return elevatorState != null; }
```

Comment:
are we using this functionality?

Reply:
Not currently, this is the logic for multiple magnets, which isn't needed at the moment. We can safely delete it if we are certain we won't be adding more magnets.

Reply:
If not using multiple magnets, you can probably delete the whole file and just use a `DigitalInput`?

Reply:
It depends if we plan to add more magnets.